frontend/src/pages/profile/components/user-articles-genres-stats/user-articles-genres-stats.tsx

```diff
@@ -61,63 +61,66 @@ const UserArticlesGenresStats: React.FC<Properties> = ({ className }) => {
   );
 
   return (
-    <div className={getValidClassNames(styles.wrapper, className)}>
+    <div className={styles.statsContainer}>
       <h3 className={styles.title}>Genres Stats</h3>
-      <div className={styles.chartWrapper}>
-        {userArticlesGenresStatsStatus === DataStatus.PENDING && (
-          <Loader isLoading type="dots" />
-        )}
-        {userArticlesGenresStatsStatus === DataStatus.FULFILLED && (
-          <>
-            <form className={styles.form} name="FiltersForm">
-              <div>
-                <Select
-                  name="articlePublishedStatus"
-                  placeholder="Choose article status..."
-                  label=""
-                  options={articleStatusOptions}
-                  control={control}
-                  errors={errors}
-                />
-              </div>
-            </form>
-            <ResponsiveContainer width="100%" height="100%">
-              <RechartsPieChart
-                width={GenresChartConfig.SIZE}
-                height={GenresChartConfig.SIZE}
-              >
-                <Pie
-                  cx="50%"
-                  cy="50%"
-                  label={false}
-                  dataKey="count"
-                  labelLine={false}
-                  data={chartData}
-                  innerRadius={GenresChartConfig.INNER_RADIUS}
-                  outerRadius={GenresChartConfig.OUTER_RADIUS}
+      <div className={getValidClassNames(styles.wrapper, className)}>
+        <div className={styles.chartWrapper}>
+          {userArticlesGenresStatsStatus === DataStatus.PENDING && (
+            <Loader isLoading type="dots" />
+          )}
```

Comment:
Can't we wrap up a chart with it?

Reply:
Sorry, I don't get it. You mean can we wrap up a chart with DataStatus.PENDING condition?

Reply:
Yep

Reply:
But we add a Loader when stats are loading and then render the data (on fulfilled. Isn't it write?

Reply:
And the Loader component on isLoading state replaces wrapped component with loader - I believe this would work the same way